cmd/limactl/main.go

```diff
@@ -188,7 +187,11 @@ func newApp() *cobra.Command {
 		newTemplateCommand(),
 		newRestartCommand(),
 	)
-	if runtime.GOOS == "darwin" || runtime.GOOS == "linux" {
+	switch runtime.GOOS {
+	case "darwin":
+		rootCmd.AddCommand(startAtLoginCommand())
+		rootCmd.AddCommand(newSudoersCommand())
+	case "linux":
 		rootCmd.AddCommand(startAtLoginCommand())
```

Comment:
Maybe these ones should be added in `main_{darwin,linux}.go`

Reply:
moved